sympy/utilities/lambdify.py

```diff
@@ -8,7 +8,12 @@
 from functools import wraps
 import inspect
 import textwrap
-
+from sympy import *
```

Comment:
`import *` should not be used in library code. Import what you need.

Reply:
okk, it won't happen again.